go/tasks/plugins/presto/config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+//go:generate pflags Config --default-var=defaultConfig
+
+import (
+	"context"
+	"net/url"
+	"time"
+
+	"github.com/lyft/flytestdlib/config"
+	"github.com/lyft/flytestdlib/logger"
+
+	pluginsConfig "github.com/lyft/flyteplugins/go/tasks/config"
+)
+
+const prestoConfigSectionKey = "presto"
+
+func URLMustParse(s string) config.URL {
+	r, err := url.Parse(s)
+	if err != nil {
+		logger.Panicf(context.TODO(), "Bad Presto URL Specified as default, error: %s", err)
+	}
+	if r == nil {
+		logger.Panicf(context.TODO(), "Nil Presto URL specified.", err)
+	}
+	return config.URL{URL: *r}
+}
+
+type RoutingGroupConfig struct {
+	Name                             string  `json:"name" pflag:",The name of a given Presto routing group"`
+	Limit                            int     `json:"limit" pflag:",Resource quota (in the number of outstanding requests) of the routing group"`
+	ProjectScopeQuotaProportionCap   float64 `json:"projectScopeQuotaProportionCap" pflag:",A floating point number between 0 and 1, specifying the maximum proportion of quotas allowed to allocate to a project in the routing group"`
+	NamespaceScopeQuotaProportionCap float64 `json:"namespaceScopeQuotaProportionCap" pflag:",A floating point number between 0 and 1, specifying the maximum proportion of quotas allowed to allocate to a namespace in the routing group"`
+}
+
+type RateLimiter struct {
+	Name         string          `json:"name" pflag:",The name of the rate limiter"`
+	SyncPeriod   config.Duration `json:"syncPeriod" pflag:",The duration to wait before the cache is refreshed again"`
+	Workers      int             `json:"workers" pflag:",Number of parallel workers to refresh the cache"`
+	LruCacheSize int             `json:"lruCacheSize" pflag:",Size of the cache"`
+	MetricScope  string          `json:"metricScope" pflag:",The prefix in Prometheus used to track metrics related to Presto"`
```

Comment:
I don't think you need this... you just create it as a const in your code

Reply:
This is part of the change for one of your earlier comments where we should make the cache/rate limiter configurable. Are there things that should be  configurable and others that shouldnt?

Reply:
Yeah, I would say you should drop Name and MetricScope from here...